.vscode/tasks.json

```diff
@@ -0,0 +1,141 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Aerie UI",
+      "type": "shell",
+      "command": "nvm use && npm run dev",
+      "detail": "Task to run just the UI for development.",
+      "problemMatcher": [
+        {
+          "background": {
+            "activeOnStart": true,
+            "beginsPattern": "  VITE",
+            "endsPattern": "➜  press h to show help"
+          },
+          "pattern": {
+            "regexp": ""
+          }
+        }
+      ],
+      "presentation": {
+        "group": "groupA"
+      }
+    },
+    {
+      "label": "Svelte Check",
+      "type": "shell",
+      "command": "nvm use && npm run check:watch",
```

Comment:
Interestingly even though I have nvm installed on my system i get this error when running `Aerie UI` task:
![image](https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/9359e1f6-1fd3-4923-aa9a-6ea43ad2f50a)

Reply:
Had to add this to settings.json https://stackoverflow.com/a/71156819 though seems like a fairly user-dependent setting. My vscode wasn't using -i when launching zsh (see https://github.com/microsoft/vscode/issues/143061). Wonder how we should handle this generally?